.github/workflows/node-ci.yml

```diff
@@ -230,7 +230,7 @@ jobs:
         # on the main Changeset "Version Packages" PR
         if: |
             github.event.pull_request.draft == false
-            && github.ref_name != 'changeset-release/main'
+            !startsWith(github.head_ref, 'changeset-release/')
```

Comment:
Maybe adding `&&` would be still needed here?

Reply:
Doh! Yes, it would!!